moped-editor/src/views/projects/projectView/ProjectComponents/MapAlertSnackbar.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { Snackbar } from "@material-ui/core";
+import { Alert } from "@material-ui/lab";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles(() => ({
+  root: {
+    // ComponentMapToolbar is 64 px tall and the alert is 48 px tall
+    // Set alert 8px from top to center vertically
+    top: "8px",
+  },
+}));
+
+const MapAlertSnackbar = ({
+  message,
+  severity,
+  isOpen,
+  snackbarProps,
+  alertProps,
```

Comment:
Saw this in a video as a way to make sure we can pass props supported by the MUI pieces of this component in the future. Let me know what y'all think about this pattern. I think that it is cool but might be overkill.